tls/s2n_connection.c

```diff
@@ -883,13 +873,12 @@ int s2n_connection_get_cipher_iana_value(struct s2n_connection *conn, uint8_t *f
 
     /* ensure we've negotiated a cipher suite */
     POSIX_ENSURE(
-        memcmp(
-            conn->secure->cipher_suite->iana_value,
-            s2n_null_cipher_suite.iana_value,
-            sizeof(s2n_null_cipher_suite.iana_value)
-        ) != 0,
-        S2N_ERR_INVALID_STATE
-    );
+            memcmp(
+                    conn->secure->cipher_suite->iana_value,
+                    s2n_null_cipher_suite.iana_value,
+                    sizeof(s2n_null_cipher_suite.iana_value))
+                    != 0,
+            S2N_ERR_INVALID_STATE);
```

Comment:
This formatting seems odd. It was more readable before, but I'm not sure what settings are really wrong.

Reply:
This is a result of AlignAfterOpenBracket: DontAlign.

The alternative is: 
```c
POSIX_ENSURE(memcmp(conn->secure->cipher_suite->iana_value,
                         s2n_null_cipher_suite.iana_value, sizeof(s2n_null_cipher_suite.iana_value))
                    != 0,
            S2N_ERR_INVALID_STATE);
```
You can see what it is doing here: its continuing the next line at the indentation width. Another alternative:
```c
POSIX_ENSURE(
            memcmp(conn->secure->cipher_suite->iana_value, s2n_null_cipher_suite.iana_value, sizeof(s2n_null_cipher_suite.iana_value)) != 0,
            S2N_ERR_INVALID_STATE);
```

And yet another:
```c
POSIX_ENSURE(
            memcmp(conn->secure->cipher_suite->iana_value,
                    s2n_null_cipher_suite.iana_value, sizeof(s2n_null_cipher_suite.iana_value))
                    != 0,
            S2N_ERR_INVALID_STATE);
```

In each case I control the number of newline (thanks to column limit 0) but not the indentation.

Reply:
Changing AlignAfterOpenBracket will be an expansive change. Is there a preferred heuristic I should follow?

Reply:
I prefer
```
POSIX_ENSURE(memcmp(conn->secure->cipher_suite->iana_value,
                         s2n_null_cipher_suite.iana_value, sizeof(s2n_null_cipher_suite.iana_value))
                    != 0,
            S2N_ERR_INVALID_STATE);
```

Or can we do
```
POSIX_ENSURE(memcmp(conn->secure->cipher_suite->iana_value, s2n_null_cipher_suite.iana_value, 
                         sizeof(s2n_null_cipher_suite.iana_value)) != 0,
            S2N_ERR_INVALID_STATE);
```
It's not the alignment I think is making it weird, but too many newlines.